.cloudbees/workflows/manual-approval_input_params.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: automation.cloudbees.io/v1alpha1
+kind: workflow
+name: man appr input params
+on:
+  workflow_dispatch:
+jobs:
+  ask-now:
+    with:
+      approvers: 0a808e26-f884-11ec-aa0a-42010a83ae55,0cb88e48-03d3-11eb-9d67-42010a83ae35
+      disallowLaunchByUser: false
+      instructions: Test instructions here
+      approvalInputs: |
+        str_req_def:
+          type: string
+          default: def1
+          required: true
+        str_no_req_def:
+          type: string
+          default: def2
+        bool_req_def:
+          type: boolean
+          default: true
+          required: true
+        bool_no_req_def:
+          type: boolean
+          default: true
+        bool_no_req_no_def:
+          type: boolean
+        num_no_req_def:
+          type: number
+          default: 9999
+        num_no_req_no_def:
+          type: number
+    timeout-minutes: 4320
+    delegates: cloudbees-io/manual-approval/.cloudbees/testing/custom-job.yml@v1
```

Comment:
Its a bit of a chicken and egg. We can't test against the `v1` tag until the tag is updated. I think we need to have this be pointed to the current sha but I don't know whether it will work. @kushicp can you test if it will work with `cloudbees-io/manual-approval/.cloudbees/testing/custom-job.yml@${{ cloudbees.scm.sha }}`